x/bounty/types/query.go

```diff
@@ -0,0 +1,40 @@
+package types
+
+// query endpoints supported by the bounty Querier
+const (
+	QueryHosts    = "hosts"
+	QueryHost     = "host"
+	QueryPrograms = "programs"
+	QueryProgram  = "program"
+	QueryFindings = "findings"
+	QueryFinding  = "finding"
+
+	DefaultLimit = 100
+)
+
+// QueryProgramsParams Params for query 'custom/bounty/programs'
+type QueryProgramsParams struct {
+	Page  int
+	Limit int
+	// TODO add filter
+}
```

Comment:
these are defined in cosmosSDK so we probably should just use that

Reply:
This is used for rest query.